tests/test_plugin_loader_edge_cases.py

```
@@ -507,6 +509,141 @@ def test_load_plugins_duplicate_plugin_names(self):
+
+    def test_get_plugin_dirs_local_directory_error(self):
+        """Test _get_plugin_dirs handles errors when creating local directory."""
+        with tempfile.TemporaryDirectory() as temp_dir:
+            plugin_type = "custom"
+
+            with patch(
+                "mmrelay.plugin_loader._get_plugin_root_dirs", return_value=[temp_dir]
+            ):
+                with patch("os.makedirs") as mock_makedirs:
+                    # Make local directory creation fail
+                    def side_effect(path, exist_ok=False):
```

Ruff (0.14.14)

[warning] Unused function argument: `exist_ok`

(ARG001)